website/source/docs/auth/aws-ec2.html.md

```diff
@@ -275,7 +275,9 @@ $ vault auth-enable aws-ec2
 #### Configure the credentials required to make AWS API calls
 
 Note: the client uses the official AWS SDK and will use environment variable or
-IAM role-provided credentials if available. The AWS credentials used require the IAM action `ec2:DescribeInstances` to be allowed.
+IAM role-provided credentials if available. The AWS credentials used require
+the IAM actions `ec2:DescribeInstances` and `iam:GetInstanceProfile` to be
+allowed.
```

Comment:
Technically, they only require `iam:GetInstanceProfile` if they are binding against IAM values. I wonder if this is important enough to mention (like, if people will be hesitant about giving out that permission if they don't need it).